app/predacons_cli/src/cli.py

```
@@ -0,0 +1,101 @@
+import predacons
+import os
+import rich
```

Ruff

`rich` imported but unused

Remove unused import: `rich`

(F401)